package.json

```diff
@@ -24,9 +24,11 @@
   "dependencies": {
     "array-flatten": "^2.1.2",
     "find-up": "^3.0.0",
+    "fs-extra": "^8.0.1",
```

Comment:
I never told you: I ended up loving fs-extra since you added it in zos. I'm seriously considering removing our own FileSystem module in zos-lib in favour of this package.